po/de.po

```diff
@@ -13832,9 +13823,9 @@ msgstr "nur nützlich für Fehlersuche"
 msgid "core.fsyncMethod = batch is unsupported on this platform"
 msgstr "core.fsyncMethod = batch wird auf dieser Plattform nicht unterstützt"
 
-#, fuzzy, c-format
+#, c-format
 msgid "could not parse bundle list key %s with value '%s'"
-msgstr "Konnte Konflikt-Blöcke in '%s' nicht parsen."
+msgstr "Konnte Bundle-Listenschlüssel '%s' mit Wert '%s' nicht parsen."
```

Comment:
in the original message the first %s is not surrounded with `'` characters, but this may be a change request for the original message and I would leave it like you translated it 👍